<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslationAbout</class>
 <widget class="QDialog" name="TranslationAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt Bitcoin Trader</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonCheckUpdates">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>CHECK_FOR_UPDATE</string>
       </property>
       <property name="text">
        <string>Check for Updates</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>289</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="accessibleName">
        <string>TRCLOSE</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="aboutBitcoinTraderGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="title">
      <string>About Qt Bitcoin Trader</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="aboutTextLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accessibleName">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {color: black; border: 1px solid gray; background: white; padding:6px}</string>
        </property>
        <property name="text">
         <string>Qt Bitcoin Trader is a free Open Source project&lt;br&gt;developed on pure C++ Qt and OpenSSL.&lt;br&gt;
If you want to help make project better please donate: &lt;a href=&quot;bitcoin:1d6iMwjjNo8ZGYeJBZKXgcgVk9o7fXcjc&quot;&gt;1d6iMwjjNo8ZGYeJBZKXgcgVk9o7fXcjc&lt;/a&gt;&lt;br&gt;
Feel free to send me recommendations and fixes to: &lt;a href=&quot;mailto:julyighor@gmail.com&quot;&gt;julyighor@gmail.com&lt;/a&gt;&lt;br&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="LogoGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>183</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string>LOGOBUTTON</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;margin:0</string>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_info">
       <property name="text">
        <string>Version: 1.07.98</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="aboutTranslationGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <property name="accessibleName">
      <string>ABOUT_TRANSLATION</string>
     </property>
     <property name="title">
      <string>About Translation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="langLabel">
        <property name="accessibleName">
         <string>TR00115</string>
        </property>
        <property name="text">
         <string>Language:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="languageField">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="accessibleName">
         <string>TR00116</string>
        </property>
        <property name="text">
         <string>Translator:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="translationAuthor">
        <property name="accessibleName">
         <string>TR00116</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {color: black; border: 1px solid gray; background: white; padding:6px}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TranslationAbout</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCheckUpdates</sender>
   <signal>clicked()</signal>
   <receiver>TranslationAbout</receiver>
   <slot>buttonCheckUpdates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createTranslation()</slot>
  <slot>buttonCheckUpdates()</slot>
 </slots>
</ui>
